swift/example_code/ddb/basics/Tests/MovieTests.swift

```diff
@@ -5,8 +5,8 @@
 import XCTest
 import Foundation
 import AWSDynamoDB
-import AWSClientRuntime
-import ClientRuntime
+//import AWSClientRuntime
```

Comment:
If we're commenting these out assuming they're not used, can they be removed?

Reply:
Yep. Done. Thanks!